worker/src/RTC/RtpRetransmissionBuffer.cpp

```diff
@@ -34,12 +34,14 @@ namespace RTC
 			return nullptr;
 		}
 
-		if (RTC::SeqManager<uint16_t>::IsSeqLowerThan(seq, this->startSeq))
+		auto* oldestItem = GetOldest();
```

Comment:
I think it would have been a bit cleaner to check whether this points to `nullptr` rather that checking for empty buffer above. `GetOldest()` contains the same check internally already.

Reply:
Done here: https://github.com/versatica/mediasoup/pull/1029/commits/595e2016addb955746486cd0852556e7b5f34be6